companies/apple.json

```diff
@@ -9,6 +9,7 @@
     "name": "Apple Distribution International",
     "address": "Hollyhill Industrial Estate\nHollyhill, Cork\nIreland",
     "email": "contactus.de@euro.apple.com",
+    "phone": "+353 21 428 4000",
```

Comment:
Please add sources for all points of contact listed in the record.

Reply:
@baltpeter I've added the source of contact, thanks for mentioning it.